daemon/mgr/network.go

```diff
@@ -209,14 +208,12 @@ func (nm *NetworkManager) EndpointCreate(ctx context.Context, endpoint *types.En
 	if sb == nil {
 		sandboxOptions, err := nm.sandboxOptions(endpoint)
 		if err != nil {
-			logrus.Errorf("failed to build sandbox options, err: %v", err)
-			return "", err
+			return "", fmt.Errorf("failed to build sandbox options, err: %v", err)
```

Comment:
This removal LGTM, while I wish we could do a little more change:

how about 
`fmt.Errorf("failed to build sandbox options: %v", err)` rather than 
`fmt.Errorf("failed to build sandbox options, err: %v", err)`?

Reply:
got it